services/web-app/layouts/layout/layout.module.scss

```diff
@@ -11,6 +11,8 @@
 @use '@carbon/react/scss/motion' as motion;
 @use '@carbon/react/scss/type' as type;
 @use '@carbon/react/scss/utilities/convert' as convert;
+@use '../../styles/variables' as variables;
+@use 'sass:map';
```

Comment:
minor: move built-in modules to top of the `@use` list.

Reply:
done